kano_settings/system/audio.py

```diff
@@ -99,13 +99,13 @@ def is_HDMI():
 
     if amixer_string.find(hdmi_string) != -1:
         # Make sure config file is up to date
-        if get_setting('Audio') != 'HDMI':
-            set_setting('Audio', "HDMI")
+        if get_setting('Audio') != _("HDMI"):
+            set_setting('Audio', _("HDMI"))
```

Comment:
But regardless, the settings file shouldn't contain any unicode as it should always have values saved in English.